src/modules/account/components/AddAccountBySecretRecovery/AddAccountBySecretRecovery.test.js

```diff
@@ -1,3 +1,5 @@
+import { cryptography } from '@liskhq/lisk-client';
+
```

Comment:
In general we always separated global imports from local imports, to easily recognise and separate the imports in order. Since we are not fully following it in Desktop, ill remove this for now.